tests/slate/test_darcy_hybridized_mixed.py

```diff
@@ -46,14 +46,10 @@ def test_darcy_flow_hybridization(degree, hdiv_family):
     sigma_h, u_h = w.split()
 
     w2 = Function(W)
-    sc_params = {'pc_type': 'fieldsplit',
-                 'pc_fieldsplit_type': 'schur',
-                 'ksp_type': 'gmres',
-                 'pc_fieldsplit_schur_fact_type': 'FULL',
-                 'fieldsplit_0': {'ksp_type': 'cg',
-                                  'pc_factor_shift_type': 'INBLOCKS'},
-                 'fieldsplit_1': {'ksp_type': 'cg',
-                                  'pc_factor_shift_type': 'INBLOCKS'}}
+    sc_params = {'mat_type': 'aij',
+                 'ksp_type': 'preonly',
+                 'pc_type': 'lu',
+                 'pc_factor_mat_solver_type': 'mumps'}
```

Comment:
Looks like an unrelated change, but whatever...

Reply:
Ah, it was related, albeit tangentially. Now that we can interpolate the boundary data, the solver does so, and the code was giving slightly different answers when solved with an iterative solver with a loose tolerance. Using a direct solver fixed this.

Reply:
I see, thanks for the explanation.